docs/schedule/best-practices.zh_TW.md

```diff
@@ -74,8 +74,9 @@ __例子：__
 
 ### routes.txt
 
-|字段名稱 |推薦 |
-| --- | --- |
+|字段名稱 | 推薦                                                                                                             |
+| --- |----------------------------------------------------------------------------------------------------------------|
+|`agency_id` | 應該包括在內，即使 Feed 中只有一家機構。 （另見建議包括agency_id在[agency.txt](#agencytxt)和[fare_attributes.txt](#fare_attributes.txt) ) |
```

Comment:
agency.txt, fare_attributes.txt not formatted as code
extra space before final bracket